models.py

```diff
@@ -109,8 +108,8 @@ class HighlightPoint(Point, Base):
         Index('highlight_long_lat_idx', 'latitude', 'longitude'),
     )
 
-    HIGHLIGHT_TYPE_USER_SEARCH = 1
-    HIGHLIGHT_TYPE_USER_GPS = 2
+    HIGHLIGHT_TYPE_USER_SEARCH = CONSTANT_HIGHLIGHT_TYPE_USER_SEARCH
```

Comment:
Why repeat the constant? Isn't it enough to just use the ones from `constants.py`?